.github/ISSUE_TEMPLATE/Standard-IFT-Issue-Template.md

```diff
@@ -0,0 +1,58 @@
+---
+name:  IFT Issue
+about: Use this template for creating manually detected Intermittent Failing Tests
+labels: Intermittently Failing Test
+---
+
+# Failed Scenario:
+< The title of the failed behat scenario. >
+
+## Error Step:
+< The error that is thrown on CircleCI for the failed step. >
+
+## CircleCI link:
+< The link to the failed build on CircleCI. >
+
+# Acceptance Criteria
+- [ ] The behat test should not fail again.
```

Comment:
I suggest remove the Behat word, or add another bullet to the Acceptance Criteria for those cases where the failing test is a Unit test. I believe your solution only works for Behat, but Unit tests also can be IFT.

Reply:
Having IFTs in Phpunit is very rare case but yeah you are right. It is possible. I will make the change.